### DISCUSSION TITLE
Stuck on KAMP_Settings.cfg does not exist

### DISCUSSION THREAD

**Discussion by steveharman** [Guilouz/Creality-Helper-Script-Wiki#456] [Issues]
Some time ago I followed the instructions here to manually add a button into the web interface which allowed me to toggle KAMP on/off.  It worked fine but the repo was always shown as "dirty" in Software Update, so today I thought I'd do away with the manual modification, now that v5.0 of the script supports this KAMP toggle feature.

I thought the safest idea would be to start from a "clean" install by removing KAMP and reinstalling it via the script.  Hoping that would remove the manual edits I'd made previously.

Sadly after completing a remove and reinstall I'm now at the point where Fluidd reports:

```
Include file '/usr/data/printer_data/config/Helper-Script/KAMP_Settings.cfg' does not exist

Once the underlying issue is corrected, use the "RESTART"
command to reload the config and restart the host software.
Printer is halted
```

I've had a look through various .cfg files but I can't find where this file is being included.  Nor can I seem to track down the post in the discussions here, which may have given me a clue.

What can I do to remedy the situation please?

**Comment by xanatos451** [comment]
Hitting the same issue myself. Not sure if this is due to a bug in a recent version, or if something else is going on that we both just hit.

**Reply by xanatos451** [reply]
@steveharman I just removed the second instance of the file `[include Helper-Script/KAMP_Settings.cfg]` in the `printer.cfg` and the error went away. I'm thinking the file may have moved around and the other path was still in our configurations. There should also be a line that says `[include Helper-Script/KAMP/KAMP_Settings.cfg]`, so make sure you leave that one there. Save and restart, and you should be good to go.

**Reply by steveharman** [reply]
T'darrrrr! That was precisely the problem for me too.  Many thanks!

![Notification_Center](https://github.com/Guilouz/Creality-K1-Series/assets/12557929/c05732d2-af84-487b-918a-1d3f24381741)
